sal_interfaces/HVAC/HVAC_Telemetry.xml

```diff
@@ -25,6 +25,13 @@
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
+    <item>
+      <EFDB_Name>timestamp</EFDB_Name>
+      <Description>Time at which the data was determined (TAI unix seconds).</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
```

Comment:
As timestamp is de-facto a unique key for telemetry data, I would recommend to put timestamp as the first member. *applies to all entries, not only this one*